style/README.md

```diff
@@ -1,14 +1,22 @@
 Style
 =====
 
-A guide for programming in style.
+Delivering valuable features
+and working software
+are our ultimate goals.
 
+This style guide should be used
+in service of those goals,
+and not as an end in itself.
```

Comment:
This last line doesn't really communicate anything to me that "should be used in service of those goals" doesn't already say.